dev/run

```diff
@@ -592,10 +599,9 @@ def set_boot_env(ctx):
 
 @log("Start node {node}")
 def boot_node(ctx, node):
-    erl_libs = os.path.join(ctx["rootdir"], "src")
     set_boot_env(ctx)
     env = os.environ.copy()
-    env["ERL_LIBS"] = os.pathsep.join([erl_libs])
+    env["ERL_LIBS"] = os.path.join(ctx["rootdir"], "src")
```

Comment:
I wasn't sure what was going on here with the `os.pathsep.join` piece, since there's only one path. It felt like a vestigial thing. I cleaned it up to match the same dance being done in the `check_boot_script` method.